### ISSUE TITLE
API /api/v1/label/[[label]]/values response strange values

### ISSUE THREAD

**Issue by cocotyty** [VictoriaMetrics/VictoriaMetrics#933]
**Describe the bug**

API /api/v1/label/[[label]]/values response strange values

**To Reproduce**
Steps to reproduce the behavior

Add federation jobs scrape from k8s prometheus instance.

Then curl  https://${VM_HOST}/api/v1/label/namespace/values .

The response like this:
```js
{
"status": "success",
"data": [
" 0 1606291716008\ncon",
"\",app_kubernet",
"\",id=\"/kubepods/bu",
"\",kub",
"\",namesp",
"\"10.112.",
"\"dm-w",
"\"kubernetes-",
"\"tikv_set02",
"\"zone",
"\"} 0 ",
",instance=\"10",
",jaeger_age",
// ...
}
```
**Expected behavior**
normal label values.
**Screenshots**
![image](https://user-images.githubusercontent.com/2241753/100702510-c4a68080-33dc-11eb-9445-58273678a166.png)

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
$ ./victoria-metrics-prod --version
victoria-metrics-20190730-121249-heads-single-node-0-g671d9e55
```
v1.44.0

**Used command-line flags**
Command-line flags are listed as `flag{name="httpListenAddr", value=":443"} 1` lines at `/metrics` page.
See the following docs for details:

* [monitoring for single-node VictoriaMetrics](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/README.md#monitoring)
* [montioring for VictoriaMetrics cluster](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/cluster/README.md#monitoring)

**Additional context**
Add any other context about the problem here such as error logs from VictoriaMetrics and Prometheus,
`/metrics` output, screenshots from the official Grafana dashboards for VictoriaMetrics:

* [Grafana dashboard for single-node VictoriaMetrics](https://grafana.com/dashboards/10229)
* [Grafana dashboard for VictoriaMetrics cluster](https://grafana.com/grafana/dashboards/11176)

**Comment by valyala** [comment]
This looks like you hit the issue similar to https://github.com/VictoriaMetrics/VictoriaMetrics/issues/903 . This issue has been fixed in v1.47.0 . Could you upgrade all the VictoriaMetrics components to this version (or to the [latest version](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) ) and verify whether `/api/v1/label/[[label]]/values` continues returning unexpected values in a day after the upgrade? Note that by default the `/api/v1/label/[[label]]/values` handler returns label values on a time range covering all the data, so it will continue returning unexpected results. Pass `start` and `end` query args to the handler in order to limit the time range for querying. See [these docs](https://prometheus.io/docs/prometheus/latest/querying/api/#querying-label-values) for details.

VictoriaMetrics rounds the time range passed to `/api/v1/label/[[label]]/values` to a day precision. So it may return unexpected label values if the specified time range intersects days when old VictoriaMetrics version was running.

**Comment by cocotyty** [comment]
Thank you  it works!
